spec/operators/catch-spec.js

```diff
@@ -0,0 +1,104 @@
+/* globals describe, it, expect */
+var Rx = require('../../dist/cjs/Rx');
+var Observable = Rx.Observable;
+
+describe('Observable.prototype.catch()', function () {
+  it('should catch the error and allow the return of a new observable to use', function (done) {
+    var expected = [1, 2, 'foo'];
+    var i = 0;
+    Observable.of(1, 2, 3)
+      .map(function (n) {
+        if (n === 3) {
+          throw 'bad';
+        }
+        return n;
+      })
+      .catch(function (err) {
+        expect(err).toBe('bad');
```

Comment:
This repeated expectation seems like it should be its own spec, and removed from other specs.